README.md

```diff
@@ -5,14 +5,18 @@ Introduction to DOM manipulation
 
 1. Clone this repo.
 
-2. Open exercise/exercise.js and follow the instructions.
+2. Open `exercise/exercise.js` and follow the instructions.
 
 ## What is the DOM?
 
-The DOM is the _Document Object Model_, it is the browser's interpretation of the page once it has been [_parsed_](https://developer.mozilla.org/en-US/docs/Glossary/Parse).
+The DOM is the _Document Object Model_. It is the browser's interpretation of the page once it has been [_parsed_](https://developer.mozilla.org/en-US/docs/Glossary/Parse).
+
+It's something implemented by web browsers rather than provided by the JavaScript language itself.
 
 ## DOM elements
 
+An _element_ is the JavaScript representation of something on the page.
+
 In your browser, if you _right-click_ on a page, you see _inspect element_.
```

Comment:
This shows `inspect` rather than `inspect element` in chrome and in the asset image bellow. Perhaps we should consider changing it for consistency?

Reply:
Yeah good point. I considered taking these screenshots out, since this bit is kind of confusing. I realised that some people may not know about the inspector at this point in the course though

Reply:
```suggestion
If you right click on a page you should see an "Inspect" option.
```

Reply:
We'll be there to help them out :) By the end of the week they'll build the todo list and they'll be extremely used to it. Thanks for editing